Content.Shared/Chat/TypingIndicator/TypingIndicatorEvents.cs

```diff
@@ -18,6 +18,15 @@ public TypingChangedEvent(bool isTyping)
     {
         IsTyping = isTyping;
     }
+
+    // DeltaV: Begin Indicator Override
+    public readonly ProtoId<TypingIndicatorPrototype>? OverrideIndicator;
+    public TypingChangedEvent(bool isTyping, ProtoId<TypingIndicatorPrototype>? proto)
+    {
+        IsTyping = isTyping;
+        OverrideIndicator = proto;
+    }
+    // DeltaV: End Indicator Override
```

Comment:
just have it default to null no reason to make a duplicate constructor

Reply:
Sorry, work habit